server/apps/content.py

```diff
@@ -22,6 +23,9 @@
 
 not_analyzed = {'type': 'string', 'index': 'not_analyzed'}
 
+pub_status = ['usable', 'withhold', 'canceled']
+PUB_STATUS = namedtuple('PUBSTATUS', ['USABLE', 'HOLD', 'CANCELED'])._make(pub_status)
```

Comment:
Hm, probably better not to use methods that are meant to be private, the same can be achieved by simply instantiating the class created by `namedtuple(...)`:

```py
namedtuple('PUBSTATUS', ['USABLE', 'HOLD', 'CANCELED'])(*pub_status)
```